src/writer.h

```diff
@@ -34,9 +35,13 @@ class FileWriter : public Writer {
     int _fd;
     char* _buf;
     size_t _size;
+    bool _keepOpen = false;
```

Comment:
This will issue a compiler warning with older gcc.
We should migrate to a newer C++ standard first, but that's a separate story.